src/controllers/controlpickermenu.cpp

```diff
@@ -475,7 +488,7 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                                effectUnitMenu, true);
             addPrefixedControl(effectUnitGroup, "super1",
                                tr("Super Knob"),
-                               tr("Super Knob (control linked effect parameters)"),
+                               tr("Super Knob (control effects' Meta Knobs)"),
```

Comment:
In English, the apostrophe is written after the "s" for plural possessive nouns.